test/Frontend/dependencies.swift

```diff
@@ -53,6 +54,48 @@
 // CHECK-IMPORT-DAG: CoreGraphics.swift
 // CHECK-IMPORT-NOT: :
 
+// CHECK-IMPORT-TRACK-SYSTEM-LABEL: - :
+// CHECK-IMPORT-TRACK-SYSTEM: dependencies.swift
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: CoreFoundation.swift
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: CoreGraphics.swift
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: Foundation.swift
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: ObjectiveC.swift
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: Inputs/dependencies/$$$$$$$$$$.h
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: Inputs/dependencies/UserClangModule.h
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: Inputs/dependencies/extra-header.h
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: Inputs/dependencies/module.modulemap
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: swift/clang/include/module.modulemap
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: swift/macosx/x86_64/Swift.swiftmodule
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: swift/shims/AssertionReporting.h
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: swift/shims/CoreFoundationShims.h
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: swift/shims/FoundationShims.h
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: swift/shims/GlobalObjects.h
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: swift/shims/HeapObject.h
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: swift/shims/KeyPath.h
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: swift/shims/LibcShims.h
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: swift/shims/RefCount.h
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: swift/shims/RuntimeShims.h
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: swift/shims/RuntimeStubs.h
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: swift/shims/SwiftStdbool.h
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: swift/shims/SwiftStddef.h
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: swift/shims/SwiftStdint.h
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: swift/shims/System.h
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: swift/shims/UnicodeShims.h
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: swift/shims/Visibility.h
+// CHECK-IMPORT-TRACK-SYSTEM-DAG: swift/shims/module.modulemap
```

Comment:
It's probably not a good idea to list all of these, since they may change. Maybe just pick one or two that are unlikely to change?